tempto-core/src/main/java/io/prestodb/tempto/Requirements.java

```diff
@@ -47,19 +47,19 @@ public static CompositeRequirement allOf(Requirement... requirements)
 
     public static CompositeRequirement allOf(List<Requirement> requirements)
     {
-        return new MultiCompositeRequirement(copyOf(transform(wrapAsCompositeRequirements(requirements), ImmutableSet::<CompositeRequirement>of)));
+        return new MultiCompositeRequirement(copyOf(wrapAsCompositeRequirements(requirements).stream().map(ImmutableSet::of).collect(Collectors.toList())));
```

Comment:
collect `toImmutableSet` then there is no need to use `ImmutableSet.copyOf`, static import. Add new line before `.map` and `.collect`.